templates/cinderapi/config/cinder-api-config.json

```diff
@@ -1,23 +0,0 @@
-{
```

Comment:
is it correct that this one is deleted? I think it needs to be updated for cinder though.

Reply:
yes it is, sorry